lib/handler/configurator/fastcgi.c

```diff
@@ -150,15 +150,15 @@ static int create_spawnproc(h2o_configurator_command_t *cmd, yoml_t *node, const
     }
 
     /* create socket */
-    if ((listen_fd = socket(AF_UNIX, SOCK_STREAM, 0)) == -1) {
+    if ((listen_fd = h2o_sysfn(socket, AF_UNIX, SOCK_STREAM, 0)) == -1) {
```

Comment:
We would change the source code of h2o so that certain syscalls will be invoked through the `h2o_sysfn` macro.